actix-web/src/sec-5-errors.md

```diff
@@ -1,10 +1,11 @@
 # Errors
 
-Actix uses the [*Error*](../../actix-web/actix_web/error/struct.Error.html) type
-and [*ResponseError*](../../actix-web/actix_web/error/trait.ResponseError.html)
-trait for error handling from web handlers.
+Actix uses its own [`actix_web::error::Error`][actixerror] type and
+[`actix_web::error::ResponseError`][responseerror] trait for error handling
+from web handlers.
```

Comment:
I made the paths on these type/traits explicit because I think that the different between `Error` exposed by actix-web and the `Error` trait from the standard library is a little bit confusing. They're mentioned in two adjacent paragraphs, and you won't catch the distinction unless you're already familiar with both. Also changed these to use backticks instead of emphasis (`**`) for consistency.

Reply:
👍